README.md

```diff
@@ -175,9 +175,9 @@ This endpoint returns JSON. An example follows below.
     "accredited_investor": true,
     "residential_address_country": "US"
   },
-  "verifications": {
-    "v1": true
-  }
+  "verifications": [
+    { level: "v1" }
```

Comment:
how about `{level: "v1", status: "..."}`?

Reply:
Right now, megalodon is filtering these to approved verifications, so `status` would always be `approved`.